cmd/vendor_update.go

```diff
@@ -37,12 +37,22 @@ kubebuilder update vendor
 `,
 		Run: func(cmd *cobra.Command, args []string) {
 			dieIfNoProject()
-			err := (&scaffold.Scaffold{}).Execute(
-				&model.Universe{},
+
+			universe, err := model.NewUniverse(
+				model.WithProjectFrom("PROJECT"),
+				model.WithoutBoilerplate,
+				// No resource at vendor update
+			)
+			if err != nil {
+				log.Fatalf("error creating scaffolding universe: %v", err)
```

Comment:
I am not sure if make sense print for the user it. 
What does it mean in the user perspective?

Reply:
What would you print then? Just `err`

Reply:
```suggestion
				log.Fatalf("error creating scaffolding: %v", err)
```
IHMO: the 'scaffolding universe'' will make sense for the end-user.